test/cpp_extensions/open_registration_extension.cpp

```diff
@@ -338,6 +344,14 @@ TORCH_LIBRARY_IMPL(aten, PrivateUse1, m) {
   m.impl("resize_", &custom_resize_);
 }
 
+void custom_cpu_fallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+  at::native::cpu_fallback(op, stack);
+}
+
+TORCH_LIBRARY_IMPL(_, PrivateUse1, m) {
+  m.fallback(torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
```

Comment:
Hmm, it might be more clear if instead you register the CPU fallback **directly** to `aten._foreach_add.List`. In the future we might not want the CPU Fallback to run for **all** unimplemented ops in this test suite - we're mostly using it here to explicitly test that the CPUFallback work for foreach ops.

Reply:
You can do that with:
```
TORCH_LIBRARY_IMPL(aten, PrivateUse1, m) {
  m.impl("_foreach_add.List", torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
```

Reply:
> You can do that with:
> 
> ```
> TORCH_LIBRARY_IMPL(aten, PrivateUse1, m) {
>   m.impl("_foreach_add.List", torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
> ```

Thank you for this example.
Already fixed :D